src/common/ui-wrapper.js

```diff
@@ -101,4 +102,13 @@ function getPreviewThumbnailConfig(mediaConfig: ProviderMediaConfigObject, seekb
   return previewThumbnailConfig;
 }
 
-export {UIWrapper};
+/**
+ * register UI manager handler
+ * @param {Function} UI - UI
+ * @returns {void}
+ */
+function registerUI(UI: Function): void {
```

Comment:
Can we make the type safer? (`UI: {UIManager,  EventType}`)

Reply:
unfortunately no, UI is a complex module object